### PR TITLE
Use cloudsdk appcfg

### LINKED ISSUE #30: Use appcfg directly from the sdk reference instead of using a maven dependency
see https://github.com/GoogleCloudPlatform/gradle-appengine-plugin/blob/master/src/main/groovy/com/google/appengine/task/appcfg/AppConfigTaskTemplate.groovy#L73

This way we don't have to update the maven dependency like we are in #29

### PR DESCRIPTION
**Pull request by loosebazooka**
Fixes #30

### CODE REVIEW

**Comment by loosebazooka** [comment]
@meltsufin @patflynn

File: pom.xml [hunk 1]
```diff
@@ -9,16 +9,11 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <appengine.target.version>1.9.34</appengine.target.version>
+    <appengine.target.version>1.9.36</appengine.target.version>
```
Comment (meltsufin) [inline pom.xml]: Do you still need this property?

**Comment by meltsufin** [comment]
The test needs to be fixed to compile.

File: pom.xml (hunk 1 above)
Comment (loosebazooka) [inline pom.xml]: nah, you're right

**Comment by loosebazooka** [comment]
weird that mine local test didn't catch this :\

**Comment by meltsufin** [comment]
:ship: it

**Pull request merged**